spec/feature_flagger/model_spec.rb

```diff
@@ -19,30 +19,66 @@ def id; 14 end
     end
 
     describe '#release!' do
-      it 'calls Control#release! with appropriated methods' do
-        expect(control).to receive(:release!).with(key, subject.id, 'feature_flagger_dummy_class')
-        subject.release!(key)
+      context 'with key as multiple arguments' do
+        it 'calls Control#release! with appropriated methods' do
+          expect(control).to receive(:release!).with(key, subject.id, 'feature_flagger_dummy_class')
+          subject.release!(*key)
+        end
+      end
+
+      context 'with key argument as an array' do
+        it 'calls Control#release! with appropriated methods' do
+          expect(control).to receive(:release!).with(key, subject.id, 'feature_flagger_dummy_class')
+          subject.release!(key)
+        end
       end
     end
 
     describe '#rollout?' do
-      it 'calls Control#rollout? with appropriated methods' do
-        expect(control).to receive(:rollout?).with(key, subject.id, 'feature_flagger_dummy_class')
-        subject.rollout?(key)
+      context 'with key as multiple arguments' do
+        it 'calls Control#rollout? with appropriated methods' do
+          expect(control).to receive(:rollout?).with(key, subject.id, 'feature_flagger_dummy_class')
+          subject.rollout?(*key)
+        end
+      end
+
+      context 'with key argument as an array' do
```

Comment:
I disagree. A method call with `([:key1, :key2])` is different from `(:key1, :key2)`. The first one is one argument containing an array of two symbols. Se second is two arguments containing one symbol each. Since it's two different method signatures, yes it must be another context. If you are not shure, comment the flatten method call on the `#rollout?` implementation and you'll see the the context with the array will fail.